### DISCUSSION TITLE
Welcome to SWE574-Fall2023-G1 Discussions!

### DISCUSSION THREAD

**Discussion by sadikkuzu** [SWE574-Fall2023-Group1/SWE574-Fall2023-G1#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dbaslan** [comment]
Great to be here! I'm taking care of the wiki and working on the frontend, let's get this bread! 😎
